scripts/circle-ci/run-circle-tests.sh

```diff
@@ -2,25 +2,28 @@
 
```

Comment:
Might be worth including a comment about the differences between internal and external that have resulted in the below separation.  If I were knew, this would be pretty hard to parse.